ci/infrastructure/scripts/infrastructure.sh

```
@@ -508,17 +511,17 @@ getPullRequestListViaCurl() {
 }
 
 getBranchListFromWorkspace() {
-  echo "$(eval $VS_LOG_DATESTAMP) INFO  [$VS_SCRIPTNAME] checking for branches and PRs for $VS_PARENT_JOB_NAME listed in workspaces.txt"
+  echo "$(eval $VS_LOG_DATESTAMP) INFO  [$VS_SCRIPTNAME] checking for branches and PRs for $VS_PARENT_JOB_NAME_FULL listed in workspaces.txt"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)